backend/src/test_fixtures.rs

```diff
@@ -74,8 +74,9 @@ impl NeighbourhoodFixture {
         }
     }
 
-    pub fn population_zones_path(&self) -> Option<String> {
+    pub fn context_data_path(&self) -> Option<String> {
         if self.is_cnt {
+            // TODO Rename this
```

Comment:
Before I change the scripts and cloudflare, do you like "context data" or any better ideas?